Add SSW DID method

baegjae opened the pull request:
This PR adds a specification to the registry for a SSW (Self Sovereign Wallet) DID method, `did:ssw:`.

Please refer to the link below for the specification.
https://sktston.github.io/ssw-did/did-method-spec.html

Thanks!
Signed-off-by: Ethan Sung <baegjae@gmail.com>


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/baegjae/did-spec-registries/pull/312.html" title="Last updated on Jun 10, 2021, 6:01 AM UTC (1753784)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec-registries/312/a87a4a5...baegjae:1753784.html" title="Last updated on Jun 10, 2021, 6:01 AM UTC (1753784)">Diff</a>

OR13 reviewed (approved):
Welcome SK Telecom! 

Thanks for your support of the Starcraft community of the years!

Nonblocking feedback:

- privacy and security sections are light, but you note some core issues and that you are working on it.

- "@context": "https://www.w3.org/ns/did/v1", does not describe "Ed25519VerificationKey2018", this was a recent change to did core, you need something like this now:

```
"@context": [
    "https://www.w3.org/ns/did/v1",
    "https://w3id.org/security/suites/ed25519-2018/v1"
  ],
  ```
  
  - I  think`"type": "did-communication",` may be out of date, see https://identity.foundation/didcomm-messaging/spec/#did-document-service-endpoint

TallTed commented:
@OR13 -- [Your second bullet](https://github.com/w3c/did-spec-registries/pull/312#pullrequestreview-680795432) contains `"@context":` without backticks, so it pings that poor user unnecessarily...

baegjae commented:
@OR13 
I'm glad that you enjoyed the sponsorship of our company. I am a big fan of SKT T1 team in League of Legends (lol).

Thank you for your specific comments.
I have updated the specification based on your feedback.

- Added `"https://w3id.org/security/suites/ed25519-2018/v1"`
- Changed to `"DIDCommMessaging"`

Thanks!

msporny reviewed (approved):
Document is light, but has basic content to allow registration.
